logic/out_transfer.js

```diff
@@ -31,15 +32,15 @@ __private.unconfirmedOutTansfers = {};
  * @class
  * @memberof logic
  * @see Parent: {@link logic}
- * @param {Database} db
  * @param {ZSchema} schema
  * @param {Object} logger
+ * @param {Storage} storage
  * @todo Add description for the params
  */
 class OutTransfer {
-	constructor(db, schema, logger) {
+	constructor(schema, logger, storage) {
```

Comment:
Same here.

Reply:
Same as before replaced storage and then removed. Will change.